Source/Core/Core/ConfigLoaders/IsSettingSaveable.cpp

```diff
@@ -72,6 +72,20 @@ bool IsSettingSaveable(const Config::Location& config_location)
       &Config::MAIN_OSD_MESSAGES.GetLocation(),
       &Config::MAIN_SKIP_NKIT_WARNING.GetLocation(),
 
+      // Main.DSP
+
+      &Config::MAIN_DUMP_AUDIO.GetLocation(),
+      &Config::MAIN_DSP_JIT.GetLocation(),
+      &Config::MAIN_DSP_CAPTURE_LOG.GetLocation(),
+      &Config::MAIN_DUMP_AUDIO_SILENT.GetLocation(),
+      &Config::MAIN_DUMP_UCODE.GetLocation(),
+      &Config::MAIN_AUDIO_BACKEND.GetLocation(),
+      &Config::MAIN_AUDIO_VOLUME.GetLocation(),
+      &Config::MAIN_AUDIO_MUTED.GetLocation(),
+#ifdef _WIN32
+      &Config::MAIN_WASAPI_DEVICE.GetLocation(),
+#endif
```

Comment:
Instead of adding all of these, you could add "DSP" to the list of sections at line 29.

Also, I think you've forgotten MAIN_DSP_HLE.

Reply:
Actually, speaking of DSP HLE, you don't seem to have removed that one from ConfigManager.cpp. Please be consistent with whether you're intending to port that one over or not. If it's not going to be in the new config system, then you shouldn't be adding calls to `Config::Get(Config::MAIN_DSP_HLE)`.

Reply:
Added DSP to line 29, thanks for that cleanup. I had forgotten MAIN_DSP_HLE, so I've added that in `s_setting_saveable` under Main.Core.

You're right, I hadn't removed it from ConfigManager.cpp but I've done so now. There was an SConfig function dependent on bDSPHLE and bDPL2Decoder, so I ported both over now.